docs/4.3/pages/aws-oss-guide.md

```diff
@@ -67,7 +67,7 @@ stores the state of the system and S3 will store audit logs.
 ![AWS Intro Image](../img/aws/aws-intro.png)
 
 ### EC2 / Autoscale
-To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions. Instructions on using these [AMIs are below](#single-oss-teleport-amis-manual-gui-setup).
+To run Teleport in a HA configuration we recommend using m4.large instances. It's best practice to separate the proxy and authentication server, using autoscaling groups for both machines. We have pre-built AMIs for both Teleport OSS and Enterprise editions.
```

Comment:
This section was removed in 4.3, so I remove reference for it. If it was moved elsewhere, then we should add correct link instead.